catalog-rest-service/src/main/java/org/openmetadata/catalog/jdbi3/TableRepository.java

```diff
@@ -1099,7 +1100,7 @@ private void updateColumns(
         updateColumnDataLength(stored, updated);
         updateTags(
             stored.getFullyQualifiedName(),
-            EntityUtil.getFieldName(fieldName, updated.getName(), "tags"),
+            EntityUtil.getFieldName(fieldName, quoteName(updated.getName()), "tags"),
```

Comment:
Please use FullyQualifiedName.build() method and make sure you are not spreading the logic of building this all over the place.